### DISCUSSION TITLE
Placeholder on a form as a badge

### DISCUSSION THREAD

**Discussion by matijarezek** [filamentphp/filament#9812] [Help] [answered]
I would love to put a status of a ticket as a placeholder on my edit or view page of that resource.
I managed to put a placeholder which is pretty straightforward but I was wondering can it be made to look like a TextCollumn with badge property on a table.

Only thing I found is that I can put HTML inside the content of a placeholder, but I couldn't find stylings that filament uses for badges.

Maybe it would be a cool addition in next update of filament if there isn't one.

Thanks in advance!

**Comment by leandrocfe** [comment]
You could use TextEntry badge in the infolist
https://filamentphp.com/docs/3.x/infolists/entries/text#displaying-as-a-badge

This?

**Reply by matijarezek** [reply]
That's kinda it yes thank you, but this is only for the view page.
Is there something like that for the edit page or I should say a form schema?

**Reply by leandrocfe** [reply, marked as answer]
You could create a [view field](https://filamentphp.com/docs/3.x/forms/fields/custom#view-fields) and render a [badge component](https://filamentphp.com/docs/3.x/support/blade-components/badge)

**Reply by matijarezek** [reply]
Thats it! Thank you so much.
